src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example4.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HiddenField">
+      <property name="ignoreConstructorParameter" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+// xdoc section -- start
+class Example4 {
+
+  private String field;
+  private String testField;
+
+  Example4(String testField) { // OK, 'testField' param doesn't hide any field
```

Comment:
Remove ok comments.

It will s because of ignoreConstructorParameter=true

Reply:
Apologies for the oversight. I've added the appropriate 'OK' comments to all examples to ensure consistency. Please review.

Reply:
Done.